valintaperusteet-service/src/main/java/fi/vm/sade/service/valintaperusteet/service/impl/ValintatapajonoServiceImpl.java

```diff
@@ -301,6 +303,19 @@ public Valintatapajono update(String oid, ValintatapajonoCreateDTO dto) {
                 LOGGER.error(String.format("Virhe tarkistaessa onko valintatapajonolle %s suoritettu sijoitteluajoa", oid), e);
             }
         }
+
+        //jonon arvoa "eiLasketaPaivamaaranJalkeen" ei saa muokata ilman rekisterinpitäjän oikeuksia jos se on jo menneisyydessä.
+        Date oldEiLasketaJalkeen = managedObject.getEiLasketaPaivamaaranJalkeen();
+        Date newEiLasketaJalkeen = konvertoitu.getEiLasketaPaivamaaranJalkeen();
```

Comment:
Sama tässä. Ehkä näitä voisi nimittää tyyliin `Valintatapajono nykyinenValintatapajono = haeValintatapajono(oid);` ja `Valintatapajono tallennettavaValintatapajono = modelMapper.map(dto, Valintatapajono.class);`?